chat-plugins/jeopardy.js

```diff
@@ -1,772 +1,725 @@
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let params = target.split(",");
+			if (params.length !== 2) return this.errorReply("You must specify the category number and question number");
+			let categoryNumber = parseInt(params[0]);
+			if (!categoryNumber || categoryNumber < 1 || categoryNumber > room.game.categoryCount) return this.errorReply(`The category must be a number between 1 and ${room.game.categoryCount}.`);
+			let questionNumber = parseInt(params[1]);
+			if (!questionNumber || questionNumber < 1 || questionNumber > room.game.questionCount) return this.errorReply(`The question must be a number between 1 and ${room.game.questionCount}.`);
+			this.sendReplyBox(room.game.getQuestion(categoryNumber - 1, questionNumber - 1));
+		},
+
+		addplayer: 'adduser',
+		adduser: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let targetUser = Users.get(target);
+			if (!targetUser) return this.errorReply("User '" + target + "' not found.");
+			if (room.game.addPlayer(targetUser)) {
+				room.game.update();
+			} else {
+				this.errorReply("Unable to add '" + target + "' to the game.");
+			}
+		},
+
+		incorrect: 'correct',
+		correct: function (target, room, user, connection, cmd) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let reply = room.game.mark(cmd === 'correct');
+			if (reply) this.errorReply(reply);
+		},
+
+		start: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let reply = room.game.start();
+			if (reply) this.errorReply(reply);
+		},
+		removeplayer: 'removeuser',
+		removeuser: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let targetUser = Users.get(target);
+			if (!targetUser) return this.errorReply(`User '${target}' not found.`);
+			if (room.game.removePlayer(targetUser)) {
+				room.game.update();
+			} else {
+				return this.errorReply(`Unable to remove '${targetUser.name}' from the game.`);
+			}
+		},
+
+		subhost: function (target, room, user) {
```

Comment:
I already added this but it doesn't show up here bc of where the comment was made